.github/workflows/upgrade.yaml

```diff
@@ -35,14 +35,24 @@ jobs:
 
       - name: Run e2e after install
         run: |
-          make test-e2e
+          # running tests matching the base release
+          git clone https://github.com/open-policy-agent/gatekeeper.git -b v${{ env.BASE_RELEASE }} --sparse --depth 1 /tmp/gatekeeper-tests-v${{ env.BASE_RELEASE }}
+          cd /tmp/gatekeeper-tests-v${{ env.BASE_RELEASE }}
+          git sparse-checkout set test/bats
+          bats -t test/bats/test.bats
 
       - name: Save logs
         if: ${{ always() }}
         run: |
           kubectl logs -n gatekeeper-system -l control-plane=controller-manager --tail=-1 > logs-${{ matrix.HELM_VERSION }}-controller-pre-upgrade.json
           kubectl logs -n gatekeeper-system -l control-plane=audit-controller --tail=-1 > logs-${{ matrix.HELM_VERSION }}-audit-pre-upgrade.json
 
+      - name: Cleanup
+        run: |
+          kubectl delete ns gatekeeper-test-playground gatekeeper-excluded-namespace || true
```

Comment:
@maxsmythe this is also added since older tests dont have the clean up code yet.